packages/react/tsconfig.json

```diff
@@ -1,16 +1,16 @@
 {
   "compilerOptions": {
-    "moduleResolution": "node",
     "baseUrl": ".",
     "declaration": true,
     "esModuleInterop": true,
-    "jsx": "react-jsx",
-    "rootDir": "./src",
-    "outDir": "./dist",
     "importHelpers": true,
+    "jsx": "react",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "outDir": "dist/esm",
```

Comment:
Assuming this is ignored by rollup?

Reply:
It aligns to our `rollup` config (has the same value as `dir`) -> [source](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/rollup.config.js#L29)